### PR TITLE
Fetch oil osset from indexer using owner

### PR DESCRIPTION
**Pull request by HoOngEe**
Now the oil asset information could be fetched from indexer using the owner's information.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -157,17 +157,27 @@ async function airdropOilTransaction(
     return [transaction, transaction.getTransferredAsset(0)];
 }
 
+async function fetchOilTracker(owner: string): Promise<H256> {
+    const utxoUrl = getConfig<string>("oil_utxo_url").toString();
```
Comment (sgkim126) [inline index.ts]: Should toString be called?

Comment (seonpyoKim) [reply index.ts]: No it doesn't have to. I tested and deleted it.

File: config/default.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
   "accounts_url": "https://husky.codechain.io/explorer/api/accounts" ,
+  "oil_utxo_url": null,
```
Comment (majecty) [inline config/default.json]: What the `oil_utxo_url` means is unclear.

IMO, adding the indexer URL and the oil's asset type to the config file is more intuitive.

**Review by majecty — CHANGES_REQUESTED** [review]

File: config/default.json (hunk 2 above)
Comment (seonpyoKim) [reply config/default.json]: Fixed

**Review by sgkim126 — APPROVED** [review]

**Review by majecty — APPROVED** [review]

**Pull request merged**